src/api/app/policies/staging/staged_request_policy.rb

```diff
@@ -0,0 +1,15 @@
+class Staging::StagedRequestPolicy < ApplicationPolicy
+  def initialize(user, record)
+    super(user, record, user_optional: true)
```

Comment:
If user is optional, then you will have to catch it being nil

Reply:
but I think I would just leave out the constructor and rely on `require_login` in the controller to catch it

Reply:
@coolo, could you review again?  please :smiley_cat: